Mathlib/Algebra/Ring/Equiv.lean

```diff
@@ -77,7 +77,7 @@ add_decl_doc RingEquiv.toMulEquiv
 /-- `RingEquivClass F R S` states that `F` is a type of ring structure preserving equivalences.
 You should extend this class when you extend `RingEquiv`. -/
 class RingEquivClass (F R S : Type*) [Mul R] [Add R] [Mul S] [Add S] [EquivLike F R S] : Prop
-  extends MulEquivClass F R S where
+  extends MulHomClass F R S where
   /-- By definition, a ring isomorphism preserves the additive structure. -/
   map_add : ∀ (f : F) (a b), f (a + b) = f a + f b
```

Comment:
Why not extends both MulHomClass and AddHomClass? We can rename RingEquivClass to MulAddHomClass too.
(answering [this comment](https://github.com/leanprover-community/mathlib4/pull/18806#discussion_r2291210552))